CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Added
+- Add `Assert<String>.containsSubstrings()` assertions
```

Comment:
i originally wanted to call it `containsAll`, but i've encountered some strange compilation errors doing so (some issues with ambiguous function calls i think)... 🤔

Reply:
done. so there is `contains(CharSequence)`, `contains(CharSequence...)` and `contains(Iterable<CharSequence>)` 👍 

(no need for `contains(Array<CharSequence>)`)